posts/2021-03-18-async-vision-doc.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: post
+title: "Building a shared vision for Async Rust"
+author: Niko Matsakis
```

Comment:
nit: probably wants to be async foundation wg

Reply:
Hmm, I feel like we've traditionally tried to write blog posts from individuals, to give them more voice. The [blog guidelines](https://forge.rust-lang.org/core/blogs.html) say:

> If, however, you are writing in an official capacity, then one of the Rust blogs would be a good fit. Note that this doesn’t mean you can’t write as an individual. Plenty of the posts on Rust’s blog are signed by individuals, and, in fact, that is the preferred option.

(Admittedly, I wrote that sentence =), but it was because that was our policy as I understood it.)

Reply:
That said, I agree this post is "from" the WG -- I rather like the "on behalf of" formulation. Let me also go look at what we've done in recent posts...

Reply:
...looks like most of the latest blog posts are from teams, with some "on behalf of".  I feel like we should pick something and update the official guidance. =) I'm going to go with 'on behalf of' for now, since that seems to mix the old style of keeping things personal with assigning from a team (I see advantages to both).

Reply:
OK, I pushed an update. Let me know what you think.